cibuildwheel/util.py

```diff
@@ -204,6 +204,7 @@ class BuildOptions(NamedTuple):
     package_dir: Path
     output_dir: Path
     build_selector: BuildSelector
+    test_selector: BuildSelector
```

Comment:
Nitpicking, once again, but this should probably move down ... `n` lines, before or after `test_command`?

Reply:
NamedTuple was designed for adding names to an API that already was using tuples, such as for return values. We probably should be using a dataclass, which does not have an order and the tuple behaviors, which would then not be possible to interact with "by order" so order wouldn't matter.

Anyway, I'll move it.

Reply:
Oh, this isn't about the order of the tuple, btw! Just about having an order in the code that's consistent with the order in the docs, and the docs having a reasonably chronological order (e.g., test options grouped, and things +- in the order in which you need them in the build process).

All for dataclasses, yes! :-) But they need 3.7, right?

Reply:
No, they have a backport for 3.6. Due to the design, that's as far back as you can go. Attrs is also an option, personally I go to that pretty quick because it's so much more powerful and I usually need something it can do. I think we are intending to drop 3.6 soonish? Though I want to see what the default versions of Python 3 on all GitHub runners are first. https://docs.github.com/en/actions/guides/building-and-testing-python#using-the-default-python-version , just in case we can drop the requirement that our action has to have the setup action first.

No changes for now, though, just a potential thought for a refactor. We don't have an external API, so the fact users can access these by order doesn't really hurt us since we are the only users and we know not to do that. :)

Reply:
Ah, didn't know that. But yes, agreed it's better. One of the satisfying things of dropping older Python versions ;-)

Reply:
> We don't have an external API, so the fact users can access these by order doesn't really hurt us since we are the only users and we know not to do that. :)

Exactly, purely nitpicking to make our future live easier when looking for that one option, switching back and forth between code and docs. Well, and consistency sometimes helps to spot bugs/weird things ;-)

Reply:
Not against nitpicking, or strict requirements. :)

Reply:
Alright. If there's nothing else to complain about, then... well, yeah, I need to start nitpicking ;-)

(Open for other things, btw; I just really like keeping an intuitive/consistent order, here, for some reason :-) )

Reply:
Moved.